### PR TITLE
Group Education - report at Patient record

### PR DESCRIPTION
**Pull request by anvmn**
#1097

### CODE REVIEW

**Comment by anvmn** [comment]
Selection_397

**Comment by anvmn** [comment]
Peek 2024-04-11 20-55

File: client/src/assets/scss/_new.scss [hunk 1]
```diff
@@ -4696,6 +4696,41 @@ div.page-activity.patient-record {
...
+    display: grid;
+    grid-gap: 20px;
+    grid-template-columns: 60% auto;
+    text-align: center;
+  }
+
+  .entry.group-education {
+    background: $color-white;
+    border-bottom: 2px solid $color-gray-border;
+    // display: flex;
```
Comment (AronNovak) [inline client/src/assets/scss/_new.scss]:
```suggestion
```

File: client/src/assets/scss/_new.scss [hunk 2]
```diff
@@ -4696,6 +4696,41 @@ div.page-activity.patient-record {
...
+    font-size: 18px;
+    font-weight: 500;
+    // padding: 20px;
```
Comment (AronNovak) [inline client/src/assets/scss/_new.scss]:
```suggestion
```

**Review by AronNovak — APPROVED** [review]

**Pull request merged**